VCProjectEngineWrapper/VCCLCompilerToolWrapper.cs

```diff
@@ -86,7 +86,24 @@ public string[] GetAdditionalIncludeDirectories()
 
 		public string[] GetPreprocessorDefinitions()
 		{
-			return _wrappedRules.GetEvaluatedPropertyValue("PreprocessorDefinitions").Split(';');
+			string additionalDefines = "";
+			switch (_wrapped.RuntimeLibrary)
+			{
+			case runtimeLibraryOption.rtMultiThreaded:
+				additionalDefines += "_MT;";
+				break;
+			case runtimeLibraryOption.rtMultiThreadedDebug:
+				additionalDefines += "_DEBUG;_MT;";
+				break;
+			case runtimeLibraryOption.rtMultiThreadedDLL:
+				additionalDefines += "_MT;_DLL;";
+				break;
+			case runtimeLibraryOption.rtMultiThreadedDebugDLL:
+				additionalDefines += "_DEBUG;_MT;_DLL;";
+				break;
+			}
+
+			return (additionalDefines + _wrappedRules.GetEvaluatedPropertyValue("PreprocessorDefinitions")).Split(';');
```

Comment:
Adds implied definitions for the current `RuntimeLibrary` setting based on [this documentation.](https://docs.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library)